src/model/ShadingControl.hpp

```diff
@@ -121,6 +129,18 @@ class MODEL_API ShadingControl : public ResourceObject {
 
   void resetSetpoint(); // TODO: makes little sense like isSetpointDefaulted
 
+  bool setGlareControlIsActive(bool glareControlIsActive);
```

Comment:
Are we sure we need this versus just deleting the field altogether?

Reply:
Seems like it allows the user to have more control of the `ShadingControl` object...

Reply:
If there are uses cases where it makes sense to switch it to Yes or No, then that's totally fine. I haven't looked at it, I'm just asking the question, based on the fact that before this field was automatically set to Yes/No in both the model API and FT.

Reply:
From what I can tell, it's required to be Yes if ShadingControlType=OnIfHighGlare (so this PR at least corrects that, since before it was hardcoded to No). Additionally, it seems like GlareControlIsActive can be Yes or No if any other ShadingControlType.

Currently I have the check for setting GlareControlIsActive to Yes when ShadingControlType=OnIfHighGlare in the forward translator. Perhaps it should instead be in the setter for GlareControlIsActive (i.e., you cannot set GlareControlIsActive to No when ShadingControlType=OnIfHighGlare)?

@jmarrec